src/sql/workbench/services/notebook/browser/models/notebookModel.ts

```diff
@@ -434,7 +434,7 @@ export class NotebookModel extends Disposable implements INotebookModel {
 					});
 				}
 				// Only add new parameter cell if notebookUri Parameters are found
-				if (notebookUriParams) {
+				if (notebookUriParams && this.notebookUri?.scheme !== 'git') {
```

Comment:
What's this for?

Reply:
A parameter cell will be injected if there are no changes made here.

Reply:
Is this actually related to this PR? How did you get this URI?

Reply:
Yes. This is what's passed into the notebook input.